packages/components/src/internal/components/editable/Cell.tsx

```diff
@@ -267,6 +270,42 @@ export class Cell extends React.PureComponent<Props, State> {
         }
     };
 
+    getRenderer = (): ReactNode => {
+        const { cellActions, col, colIdx, rowIdx, values, row } = this.props;
+
+        const renderer = resolveRenderer(col);
+
+        if (renderer) {
+            const onQSChange = (name: string, value: string | any[], items: any) => {
+                cellActions.modifyCell(
+                    colIdx,
+                    rowIdx,
+                    [{ raw: items?.value, display: items?.label }],
+                    MODIFICATION_TYPES.REPLACE
+                );
```

Comment:
there looks to be a few other places in this file that have the same block of code. any chance they could be refactored to share an implementation that passes in the raw and display values?

Reply:
Good call, factored that into a function